src/renderer/pages/project/script_editor/reference.vue

```
@@ -30,4 +80,81 @@
+    const res = (await window.electronAPI.mulmoHandler(
+      "mulmoReferenceImageFetchURL",
+      props.projectId,
+      imageKey,
+      mediaUrl.value,
+    )) as { result: boolean; imageType: string; path: string };
+    if (res.result) {
+      emit("updateImagePath", imageKey, "./" + res.path);
+      mediaUrl.value = "";
+      loadReference();
+
+    }
```

ESLint

[error] Delete `⏎`

(prettier/prettier)